File: Quizify-FrontEnd/src/app/pclasses/game.ts
import { Category } from './category';
import { Question } from './question';
import { Admin } from './admin';
import { Topic } from './topic';

export class Game {

     id:string;
     name:string;
     category:Category;
     playcount:number;
      questions:Question[];
     admin:Admin;
     topic:Topic;
     level:string;
}

File: game-manager-service/src/main/java/com/stackroute/quizify/gamemanager/domain/Game.java
package com.stackroute.quizify.gamemanager.domain;


import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;


@Data
@Document(collection="games")
public class Game {

    @Id
    private long id;
    private String name;
    private Category category;
    private Topic topic;
    private int playCount;
    private int numOfQuestion;
    private List<Question> questions;
    private Admin admin;
    private String level;


//    @Override
//    public String toString() {
//        return "Game{" +
//                "id='" + id + '\'' +
//                ", name='" + name + '\'' +
//                ", category=" + category +
//                ", playcount=" + playcount +
//                ", quesNum=" + quesNum +
//                ", questions=" + questions +
//                ", admin=" + admin +
//                ", topic=" + topic +
//                ", level='" + level + '\'' +
//                '}';
//    }

}
